test/car_req_test.exs

```diff
@@ -724,7 +724,7 @@ defmodule CarReqTest do
       )
 
       assert_receive {:event, [:http_car_req, :request, :stop], _,
-                      %{reason: %Mint.TransportError{reason: :timeout}}}
```

Comment:
We would need to update some parts of the code that use this reference, mostly tests and type specs